contracts/UniPriceOracle.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.28;
```

Comment:
For symmetry, let's have such a wrapper. The gas is cheap on these chains, makes it just easier.